operators/deploy/crds/crownlabs.polito.it_workspaces.yaml

```diff
@@ -52,6 +52,13 @@ spec:
               prettyName:
                 description: The human-readable name of the Workspace.
                 type: string
+              autoEnroll:
+                description: AutoEnroll capability definition. If omitted, no autoenroll features will be added.
+                enum:
+                - none
+                - withApproval
+                - self
```

Comment:
Would suggest something more like `immediate` instead of `self`. @frisso got any suggestion?